### PR TITLE
update tr translate

### PR DESCRIPTION
**Pull request by Ylarod**

### CODE REVIEW

**Comment by Ylarod** [comment]
@mdksec

**Comment by mdksec** [comment]
I took the english file and translated it. I couldn't find anything else broken except translations

**Comment by Ylarod** [comment]
> I took the english file and translated it. I couldn't find anything else broken except translations



> I took the english file and translated it. I couldn't find anything else broken except translations

quote is not escaped

**Review by symbuzzer — COMMENTED** [review]
Many translations are wrong :(

**Comment by mdksec** [comment]
> Many translations are wrong :(

What's wrong? Tell me and we'll fix it. many places in your translation are in English

**Comment by dabao1955** [comment]
Why not try using [weblate](https://hosted.weblate.org/projects/kernelsu/manager/) for translation?

**Comment by Ylarod** [comment]
> [strings.xml.txt](https://github.com/tiann/KernelSU/files/13251740/strings.xml.txt) I made a few changes, latest

updated

**Comment by mdksec** [comment]
> Why not try using [weblate](https://hosted.weblate.org/projects/kernelsu/manager/) for translation?

I had to edit it all from the beginning, so it was easier this way. Once the app is updated, I'll check it again and change anything that looks bad

**Comment by dabao1955** [comment]
> > Why not try using [weblate](https://hosted.weblate.org/projects/kernelsu/manager/) for translation?
> 
> I had to edit it all from the beginning, so it was easier this way. Once the app is updated, I'll check it again and change anything that looks bad

Although, why should I edit all the content from the beginning? Isn't it easier to edit the wrong translation content? And this project is using this method to manage translation instead of submitting [PR](#794) alone(

**Comment by mdksec** [comment]
> > > Why not try using [weblate](https://hosted.weblate.org/projects/kernelsu/manager/) for translation?
> > 
> > I had to edit it all from the beginning, so it was easier this way. Once the app is updated, I'll check it again and change anything that looks bad
> 
> Although, why should I edit all the content from the beginning? Isn't it easier to edit the wrong translation content? And this project is using this method to manage translation instead of submitting [PR](https://github.com/tiann/KernelSU/pull/794) alone(

I'm going to edit some parts of the translation in a bit. It's the best translation in the world man

**Comment by mdksec** [comment]
[strings.xml.txt](https://github.com/tiann/KernelSU/files/13256123/strings.xml.txt)

**Comment by dabao1955** [comment]
> > > > Why not try using [weblate](https://hosted.weblate.org/projects/kernelsu/manager/) for translation?
> > > 
> > > 
> > > I had to edit it all from the beginning, so it was easier this way. Once the app is updated, I'll check it again and change anything that looks bad
> > 
> > 
> > Although, why should I edit all the content from the beginning? Isn't it easier to edit the wrong translation content? And this project is using this method to manage translation instead of submitting [PR](https://github.com/tiann/KernelSU/pull/794) alone(
> 
> I'm going to edit some parts of the translation in a bit. It's the best translation in the world man

Uh-huh, don't you like using weblate for translation?

**Comment by mdksec** [comment]
> > > > > Why not try using [weblate](https://hosted.weblate.org/projects/kernelsu/manager/) for translation?
> > > > 
> > > > 
> > > > I had to edit it all from the beginning, so it was easier this way. Once the app is updated, I'll check it again and change anything that looks bad
> > > 
> > > 
> > > Although, why should I edit all the content from the beginning? Isn't it easier to edit the wrong translation content? And this project is using this method to manage translation instead of submitting [PR](https://github.com/tiann/KernelSU/pull/794) alone(
> > 
> > I'm going to edit some parts of the translation in a bit. It's the best translation in the world man
> 
> Uh-huh, don't you like using weblate for translation?

I really don't like the Weblate interface, it is too complicated for me. Crowdin would be better

**Comment by symbuzzer** [comment]
> > Many translations are wrong :(
> 
> What's wrong? Tell me and we'll fix it. many places in your translation are in English

You translated "superuser" as "yönetici" which means "administrator/manager". In the entire Unix community, superuser means superuser.

What are "Kurtarma Moduna Yeniden Başlat, Önyükleyiciye Yeniden Başlat, İndirme Moduna Yeniden Başlat, EDL Moduna Yeniden Başlat"? Is there such a sentence structure in Turkish?

"KernelSU\'yu nasıl yükleyip modüller kullanacağınızı öğrenin" This sentence is inverted and has a grammatical error.

You should have written "Modülleri sök" instead of "Modülleri çıkart". It could have been much funnier.

I'm sorry my friend, but the translation contains a lot of mistakes like this. I only counted some of them. Please withdraw this PR and let's work together on weblate.

**Comment by mdksec** [comment]
> > > Many translations are wrong :(
> > 
> > 
> > What's wrong? Tell me and we'll fix it. many places in your translation are in English
> 
> You translated "superuser" as "yönetici" which means "administrator/manager". In the entire Unix community, superuser means superuser.
> 
> What are "Kurtarma Moduna Yeniden Başlat, Önyükleyiciye Yeniden Başlat, İndirme Moduna Yeniden Başlat, EDL Moduna Yeniden Başlat"? Is there such a sentence structure in Turkish?
> 
> "KernelSU'yu nasıl yükleyip modüller kullanacağınızı öğrenin" This sentence is inverted and has a grammatical error.
> 
> You should have written "Modülleri sök" instead of "Modülleri çıkart". It could have been much funnier.
> 
> I'm sorry my friend, but the translation contains a lot of mistakes like this. I only counted some of them. Please withdraw this PR and let's work together on weblate.

ok, you might be right in some places. check the current translation, I think it's the best one for us. change the bad looking parts again and post it here and we'll agree. 

[strings.xml.txt](https://github.com/tiann/KernelSU/files/13256579/strings.xml.txt)

**Comment by mdksec** [comment]
If this is ok with you, approve and send. It doesn't matter if the translation is still registered in your name.

**Comment by symbuzzer** [comment]
> If this is ok with you, approve and send. It doesn't matter if the translation is still registered in your name

I think it will be more efficient over Weblate. We cannot see the original text from the XML file. It doesn't matter if the translations are registered in my name. When you translate via Weblate, it is registered in your name.

**Comment by mdksec** [comment]
> > If this is ok with you, approve and send. It doesn't matter if the translation is still registered in your name
> 
> I think it will be more efficient over Weblate. We cannot see the original text from the XML file. It doesn't matter if the translations are registered in my name. When you translate via Weblate, it is registered in your name.

Let's keep it like this for now. I think the xml file I sent can be saved to weblate and we can continue from there.If you don't approve, your translation can stay. Or leave it like this and add my translations on weblate

**Comment by dabao1955** [comment]
> > > > > > Why not try using [weblate](https://hosted.weblate.org/projects/kernelsu/manager/) for translation?
> > > > > 
> > > > > 
> > > > > I had to edit it all from the beginning, so it was easier this way. Once the app is updated, I'll check it again and change anything that looks bad
> > > > 
> > > > 
> > > > Although, why should I edit all the content from the beginning? Isn't it easier to edit the wrong translation content? And this project is using this method to manage translation instead of submitting [PR](https://github.com/tiann/KernelSU/pull/794) alone(
> > > 
> > > 
> > > I'm going to edit some parts of the translation in a bit. It's the best translation in the world man
> > 
> > 
> > Uh-huh, don't you like using weblate for translation?
> 
> I really don't like the Weblate interface, it is too complicated for me. Crowdin would be better

oh

**Comment by mdksec** [comment]
strings.xml.txt

@Ylarod update please

**Comment by xiaocenter** [comment]
https://android.googlesource.com/kernel/common/+/refs/heads/android12-5.10-2023-11   已更新内核

**Comment by symbuzzer** [comment]
@Ylarod Can we clarify whether translations will be via weblate or not?

**Comment by Ylarod** [comment]
> @Ylarod Can we clarify whether translations will be via weblate or not?

Weblate is recommended, but pr is also accepted personally.
You can move this to weblate and I will close this pr

File: manager/app/src/main/res/values-tr/strings.xml [hunk 1]
```diff
@@ -27,80 +28,80 @@
     <string name="install">Yükle</string>
     <string name="reboot">Cihazı yeniden başlat</string>
     <string name="settings">Ayarlar</string>
-    <string name="reboot_userspace">Hızlı yeniden başlat</string>
-    <string name="reboot_recovery">Kurtarma modunda başlat</string>
-    <string name="reboot_bootloader">Bootloader modunda başlat</string>
-    <string name="reboot_download">İndirme modunda başlat</string>
-    <string name="reboot_edl">EDL modunda başlat</string>
+    <string name="reboot_userspace">Hızlıca yeniden başlat</string>
```
Comment (symbuzzer) [inline manager/app/src/main/res/values-tr/strings.xml:31]: Hızlı yeniden başlat

File: manager/app/src/main/res/values-tr/strings.xml [hunk 2]
```diff
@@ -27,80 +28,80 @@
...
     <string name="home_learn_kernelsu_url">https://kernelsu.org/guide/what-is-kernelsu.html</string>
-    <string name="home_click_to_learn_kernelsu">KernelSU\'yu nasıl kuracağınızı ve modülleri nasıl kullanacağınızı öğrenin</string>
-    <string name="home_support_title">Bizi destekle</string>
-    <string name="home_support_content">KernelSU ücretsiz ve açık kaynaktır ve her zaman öyle kalacaktır. Ancak bağış yaparak bize değer verdiğinizi gösterebilirsiniz.</string>
-    <string name="about_source_code"><![CDATA[%1$s kaynak kodunu görüntüle<br/>%2$s kanalımıza katılın]]></string>
-    <string name="profile">Uygulama profili</string>
+    <string name="home_click_to_learn_kernelsu">KernelSU\'nun nasıl kurulacağını ve modüllerin nasıl kullanılacağını öğrenin</string>
+    <string name="home_support_title">Bizi destekleyin</string>
+    <string name="home_support_content">KernelSU ücretsiz ve açık kaynaklı bir yazılımdır ve her zaman öyle kalacaktır. Ancak, bize bağış yaparak destek olduğunuzu gösterebilirsiniz.</string>
+    <string name="about_source_code"><![CDATA[Kaynak kodunu görüntüle: %1$s<br/>Bizim %2$s kanalımıza katılın]]></string>
```
Comment (symbuzzer) [inline manager/app/src/main/res/values-tr/strings.xml:55]: Bizim kanalımıza değil kanalımıza olmalı

File: manager/app/src/main/res/values-tr/strings.xml [hunk 3]
```diff
@@ -27,80 +28,80 @@
...
+    <string name="require_kernel_version">Mevcut KernelSU sürümü %d, yöneticinin düzgün çalışabilmesi için çok düşük. Lütfen %d veya daha yüksek bir sürüme güncelleyin!</string>
+    <string name="settings_umount_modules_default">Varsayılan olarak modüllerin bağlantısını kes</string>
+    <string name="settings_umount_modules_default_summary">Uygulamalar için "Modüllerin bağlantısını kes" seçeneği için varsayılan değer. Etkinleştirilirse, profil ayarı yapılmamış uygulamalar için modüllerin sistemde yaptığı tüm değişiklikler kaldırılacaktır.</string>
+    <string name="profile_umount_modules_summary">Bu seçeneği etkinleştirmek, KernelSU\'nun bu uygulama için modüller tarafından değiştirilen dosyaları geri yüklemesine izin verir.</string>
+    <string name="profile_selinux_domain">İsim alanı</string>
     <string name="profile_selinux_rules">Kurallar</string>
     <string name="module_update">Güncelle</string>
     <string name="module_downloading">Modül indiriliyor: %s</string>
-    <string name="module_start_downloading">İndirme başladı: %s</string>
+    <string name="module_start_downloading">İndirmeye başlanıyor: %s</string>
```
Comment (symbuzzer) [inline manager/app/src/main/res/values-tr/strings.xml:78]: İndirme başladı olarak kalmalı

File: manager/app/src/main/res/values-tr/strings.xml [hunk 4]
```diff
@@ -1,23 +1,24 @@
...
-    <string name="home_selinux_status">SELinux Durumu</string>
+    <string name="home_unsupported_reason">KernelSU şu an sadece GKI çekirdeklerini destekliyor</string>
+    <string name="home_kernel">Çekirdek</string>
+    <string name="home_manager_version">Uygulama sürümü</string>
+    <string name="home_fingerprint">Parmak izi</string>
+    <string name="home_selinux_status">SELinux durumu</string>
     <string name="selinux_status_disabled">Devre dışı</string>
-    <string name="selinux_status_enforcing">Enforcing</string>
-    <string name="selinux_status_permissive">Permissive</string>
+    <string name="selinux_status_enforcing">Etkin</string>
```
Comment (symbuzzer) [inline manager/app/src/main/res/values-tr/strings.xml:18]: Parantez içinde Enforcing yazılabilir.

File: manager/app/src/main/res/values-tr/strings.xml [hunk 5]
```diff
@@ -1,23 +1,24 @@
...
+    <string name="selinux_status_permissive">Boşta</string>
```
Comment (symbuzzer) [inline manager/app/src/main/res/values-tr/strings.xml:19]: Parantez içinde Permissive yazılabilir.

**Review by symbuzzer — CHANGES_REQUESTED** [review]
I confirm that it is compatible with the changes I have added as a note. @mdksec

**Comment by tiann** [comment]
@symbuzzer You can use the "suggest change" feature, it makes us modify it easily:

image

**Review by symbuzzer — APPROVED** [review]
Seems good, thanks for your efforts @mdksec @Ylarod

**Review by tiann — APPROVED** [review]

**Comment by cachiusa** [comment]
Great, now admins have to go to Weblate and fix the conflict

**Comment by tiann** [comment]
So that is why we prefer weblate, once you modify the translation both sides, the conflicts will occur.

**Comment by symbuzzer** [comment]
> @Ylarod Can we clarify whether translations will be via weblate or not?


@tiann I already asked. A few months ago, my translation PR was rejected due to Weblate.

**Pull request merged**